lib/matplotlib/axes/_base.py

```diff
@@ -1656,9 +1656,12 @@ def get_aspect(self):
         """
         Return the aspect ratio of the Axes scaling.
 
-        This is either "auto" or a float giving the ratio of y/x-scale.
+        This is either "auto", "equal" or a float giving the ratio of y/x-scale.
         """
-        return self._aspect
+        if self._aspect:
+            return self._aspect
+        aspect = mpl.rcParams['axes.aspect']
+        return aspect if aspect != 'equal' else 1
```

Comment:
Let's not change anything here. rcParams lookup should be in `__init__`. Normalization "equal" -> 1 can be done in `validate_aspect`, so that "equal" is only public API but never internal state.